instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/routes/RoutesProvider.kt

```diff
@@ -80,8 +80,30 @@ object RoutesProvider {
     fun dc_short_with_alternative(context: Context): MockRoute {
         val jsonResponse = readRawFileText(context, R.raw.route_response_dc_short_with_alternative)
         val coordinates = listOf(
-            Point.fromLngLat(-77.033991, 38.891121),
-            Point.fromLngLat(-77.030023, 38.895433)
+            Point.fromLngLat(-77.033625, 38.891164),
```

Comment:
Won't it break old tests? We might use the hardcoded coordinates somewhere...

Reply:
This route is used only in reroute and preview tests. The `dc_very_short` variant is much more popular but that's not the one I updated.